Externals/imgui/imgui.vcxproj

```diff
@@ -15,6 +15,12 @@
     <Import Project="$(VSPropsDir)ClDisableAllWarnings.props" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <!--IM_ASSERT is defined as Dolphin's ASSERT() :( (DolphinLib circular dependency)-->
+      <AdditionalIncludeDirectories>$(CoreDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+    </ClCompile>
+  </ItemDefinitionGroup>
```

Comment:
See #10375. Is this actually a circular dependency? I don't think common depends directly on imgui, so it should still be acyclic (though it is still a library depending on Dolphin).

Reply:
i guess it depends on your viewpoint. technically i dont think it's circular, but it is not great to have externals including dolphin code